kmk/modules/sticky_mod.py

```diff
@@ -0,0 +1,68 @@
+from kmk.keys import make_argumented_key
+from kmk.modules import Module
+
+
+class StickyModMeta:
+    def __init__(self, kc, mod):
+        self.kc = kc
+        self.mod = mod
+
+
+class StickyMod(Module):
+    def __init__(self):
+        self._active = False
+        self._active_key = None
+        make_argumented_key(
+            names=('SM',),
+            validator=StickyModMeta,
+            on_press=self.sm_pressed,
+            on_release=self.sm_released,
+        )
+
+    def during_bootup(self, keyboard):
+        return
+
+    def before_matrix_scan(self, keyboard):
+        return
+
+    def process_key(self, keyboard, key, is_pressed, int_coord):
+        if self._active:
+            # release previous key if any other key is pressed
+            if self._active_key is not None:
```

Comment:
Is `self._active` even necessary when `self._current_key is not None` has the same effect?